python/setup.py

```diff
@@ -46,6 +46,7 @@
         "Programming Language :: Python :: 3.6",
         "Programming Language :: Python :: 3.7",
         "Programming Language :: Python :: 3.8",
+        "Programming Language :: Python :: 3.9",
     ],
     # Include the separately-compiled shared library
     setup_requires=["Cython"],
```

Comment:
Is this still needed given we are using [`build-system` in `pyproject.toml`]( https://github.com/NVIDIA/NVTX/blob/3c98c8425b0376fd8653aac7cfc6a864f3897752/python/pyproject.toml#L21-L22 )?

Reply:
Fix'd